### DISCUSSION TITLE
PositionHlCommander and MotionCommander problem and Crazyflie crash

### DISCUSSION THREAD

**Discussion by danielcmdeveloper** [bitcraze/discussions#1629] [General]
Good morning,
We are trying to test the 1 Crazyflie movement examples based on the example : position_commander_demo.py y con 

Initially, we have tried to run this simple functionality (**take_off_simple(scf)**)

```
def take_off_simple(scf):
    with MotionCommander(scf, default_height=DEFAULT_HEIGHT) as mc:
        time.sleep(3)
        mc.stop()
```

In this case, the drone takes off and starts a drift, we understand that this may be normal because at no time we are controlling the position, if it is true that we would like to take off and stay in the same place of takeoff performing a stationary flight, but we see that little by little the drone is drifting and leaves the square established with the Loco Positioning System.

On the other hand, we have tested this other functionality (**slightly_more_complex_usage()**):

```
def slightly_more_complex_usage():
    with SyncCrazyflie(uri, cf=Crazyflie(rw_cache='./cache')) as scf:
        with PositionHlCommander(
                scf,
                x=0.0, y=0.0, z=0.0,
                default_velocity=0.3,
                default_height=0.5,
                controller=PositionHlCommander.CONTROLLER_PID) as pc:
            # Go to a coordinate
            pc.go_to(1.0, 1.0, 1.0)

            # Move relative to the current position
            pc.right(1.0)

            # Go to a coordinate and use default height
            pc.go_to(0.0, 0.0)

            # Go slowly to a coordinate
            pc.go_to(1.0, 1.0, velocity=0.2)

            # Set new default velocity and height
            pc.set_default_velocity(0.3)
            pc.set_default_height(1.0)
            pc.go_to(0.0, 0.0)
```

In this second case, if you are controlling the position and you are ordering the drone to move to certain positions until it finally lands. The result we have obtained is that the drone makes a very violent movement, leaving the flight zone that controls the position and ends up losing the route and crashing.

We do not know what may be the problem and if someone else is also happening, but we would like to know where we can go investigating.
We are interested in performing flight from the Python API with controlled movements based on the position of the drone.

Thank you very much in advance.
Best regards
Daniel Caballero

**Comment by Bananazooka** [comment]
Hello Daniel, 
I think your problem was solved here #1639 .

**Comment by danielcmdeveloper** [comment]
Hola @Bananazooka ,
Yes, both posts are mine XD.

Thank you very much for your help.
Regards
:-)))
